src/main/java/org/opensearch/performanceanalyzer/collectors/NodeStatsFixedShardsMetricsCollector.java

```diff
@@ -34,7 +34,10 @@
 import org.opensearch.performanceanalyzer.util.Utils;
 
 /**
- * This collector collects metrics for fixed number of shards on a node in a single run. These
+ * Note: 'NodeStatsAllShardsMetricsCollector' is already released and out of shadow mode, this class
```

Comment:
Any reason why we cannot remove this as part of current release?

Reply:
Following the standard practice of marking the class deprecated in (n-1) release and finally removing in nth release.